### DISCUSSION TITLE
Is it possible to get comfyui to do audio like other programs can?

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [comfyanonymous/ComfyUI#6152] [General]
UVR5 has no workflows and isn't maintained anymore https://github.com/Anjok07/ultimatevocalremovergui/discussions/689

webui-tts is better but still no workflows https://github.com/rsxdalv/tts-generation-webui

does comfyui plan on supporting audio models?

**Comment by ltdrdata** [comment]
I don't know what you're looking for, but if you search for "audio" or "sound," you can find dozens of custom nodes.

![image](https://github.com/user-attachments/assets/d36b6c37-a3ff-4d61-865f-409551772027)
![image](https://github.com/user-attachments/assets/731b7c15-6626-4966-b282-528fcf394bd7)
![image](https://github.com/user-attachments/assets/8d454cd1-f60a-49eb-97de-5f2807420e6f)

Also, ComfyUI already supports an audio model called `Stable Audio` natively.
https://comfyanonymous.github.io/ComfyUI_examples/audio/

**Reply by RustoMCSpit** [reply]
im looking for stem splitting, voice conversion, etc.

**Reply by ltdrdata** [reply]
> im looking for stem splitting, voice conversion, etc.

It seems that you are looking this.
https://github.com/christian-byrne/audio-separation-nodes-comfyui
